docs/install/operator/knative-with-operators.md

```diff
@@ -3,6 +3,12 @@
 Knative provides a [Kubernetes Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) to install, configure and manage Knative.
 You can install the Serving component, Eventing component, or both on your cluster.
 
+The following table describes the supported versions of Serving and Eventing for the Knative Operator:
+
+| Operator             | Serving                                                                                                                       | Eventing                                                                                                                         |
+|----------------------|-------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|
+| v1.5.0               | v1.5(v1.5.0)<br/>v1.4(v1.4.0)<br/>v1.3(v1.3.0, v1.3.1 and v1.3.2)<br/>v1.2(v1.2.0, v1.2.1, v1.2.2, v1.2.3, v1.2.4 and v1.2.5) | v1.5(v1.5.0)<br/>v1.4(v1.4.0 and v1.4.1)<br/>v1.3(v1.3.0, v1.3.1, v1.3.2 and v1.3.3)<br/>v1.2(v1.2.0, v1.2.1, v1.2.2 and v1.2.3) |
```

Comment:
```suggestion
| v1.5.0               | v1.5.0<br/>v1.4.0<br/>v1.3.0, v1.3.1 and v1.3.2<br/>v1.2.0, v1.2.1, v1.2.2, v1.2.3, v1.2.4 and v1.2.5 | v1.5.0<br/>v1.4.0 and v1.4.1<br/>v1.3.0, v1.3.1, v1.3.2 and v1.3.3<br/>v1.2.0, v1.2.1, v1.2.2 and v1.2.3 |
```
I think I'd just keep the stuff inside the parens, as I think it might be a little easier to read. @snneji @abrennan89 thoughts?

<img width="454" alt="image" src="https://user-images.githubusercontent.com/44698588/173406558-d8e21d94-42db-42ab-ba7a-2b1b58eaf858.png">